addons/crm/views/crm_lead_views.xml

```diff
@@ -401,8 +401,10 @@
             <field name="arch" type="xml">
                 <form>
                     <group>
-                        <field name="name"/>
-                        <field name="partner_id" widget="res_partner_many2one" context="{'res_partner_search_mode': 'customer', 'show_vat': True}"/>
+                        <field name="partner_id" widget="res_partner_many2one" context="{'res_partner_search_mode': 'customer', 'show_vat': True}" string='Organization / Contact'/>
+                        <field name="name" placeholder="Product Pricing" />
+                        <field name="email_from" readonly="0" string="Email" />
```

Comment:
Why those readonly="0" ? Normally email and phone are editable and works like related on partner. Could you check ?